src/main/java/com/intuit/graphql/orchestrator/schema/transform/FederationTransformerPreMerge.java

```diff
@@ -77,6 +79,7 @@ public XtextGraph transform(XtextGraph source) {
                     .collect(Collectors.toList());
 
                     validateFieldDefinitions(source, typeDefinition);
+                    log.info("Service {} creating entity {}", source.getServiceProvider().getNameSpace(), typeDefinitionName);
```

Comment:
I am thinking this should be logged as debug and we can set it to true on data api logback file. also, we can accept a flag which when enabled logs debug log.